### PR TITLE
fixed the homepage image and the faq page

### PR DESCRIPTION
**Pull request by randal-sean-harrison**
Fixed the homepage image according to our (Don/Randy) discussion and updated the faq page to fix the rotated plus sign.

### CODE REVIEW

File: webui/static/main.css [hunk 1]
```diff
@@ -56,14 +56,38 @@ h4 {
...
+  #jumbotron {
+    background-image: url("banner-1300.jpg");
+    box-sizing: border-box;
+    font-style: normal;
+    background-size: auto;
+    background-position: top center !important;
+    background-repeat: no-repeat;
+    min-height: 350px;
+  }
+} */
```
Comment (dbrower) [inline webui/static/main.css:90]: Did you mean for this to be commented out?

File: webui/templates/home.html [hunk 2]
```diff
@@ -2,7 +2,9 @@
 
 {% block body %}
 <!-- jumbotron -->
-  <div class="jumbotron" id="jumbotron"></div>
+  <div class="jumbotron bg-nd-blue d-flex justify-content-center" id="jumbotron">
+  <img class="img-fluid" src="../static/banner-1300.jpg" alt="">
```
Comment (dbrower) [inline webui/templates/home.html:6]: use the `{{ url_for("static", ...... }}` thing here instead of hardcoding a path to the banner.

**Comment by randal-sean-harrison** [comment]
Yes, meant to leave the commented thing just for the moment until we’re satisfied with the browser solution. 

Can you adjust the url — I tried using the templating {{}} but I broke it several times. Couldn’t get it to work. Could you help with that?


Cheers,
Randy

Randal Sean Harrison, Ph.D.
Associate Librarian, Emerging Technologies
Hesburgh Libraries

Concurrent Teaching Professor, Management & Organization
Mendoza College of Business

University of Notre Dame
158 Hesburgh Library
Notre Dame, IN 46556

(574) 631-0312
***@***.*** ***@***.***>
randalseanharrison.com <https://randalseanharrison.com/>

⌖  Map to my office <https://randalseanharrison.com/img/map-to-my-office.svg>
⌖  Meet with me <https://calendly.com/randal-sean-harrison/consultation>

**Comment by dbrower** [comment]
Sure. I'll merge this and then commit an update for the url.

**Pull request merged**